meetings/2021-01-07.md

```diff
@@ -0,0 +1,41 @@
+# 2021-01-07 3rd meeting
+
+## Attendees
+
+* Aaron Stannard
+* Claire Novotny
+* Eriawan Kusumawardhono
+* Eric Erhardt
+* Immo Landwerth
+* James Jackson-South
+* John Callaway
+* Ken Cutt
+* Sébastien Ros
+* Shaun Walker
+* Tanner Gooding
```

Comment:
Suggestion for future meetings: It would be nice if the GitHub username of each attendee was also included here. In some cases the GitHub username is more known to people than their actual legal name. e.g. No idea who "James Jackson-South" is, but I know who JimBobSquarePants is :smile:

Reply:
Fixed in 10a8cee.